mslib/msui/viewwindows.py

```diff
@@ -173,10 +178,20 @@ def setFlightTrackModel(self, model):
         """
         Set the QAbstractItemModel instance that the view displays.
         """
-        self.waypoints_model = model
+        super().setFlightTrackModel(model)
+
         if self.mpl is not None:
             self.mpl.canvas.set_waypoints_model(model)
 
+            # Update Top View flighttrack name
```

Comment:
Same as above. I would prefer using members/methods for this instead of "parsing" the text. Not very maintainable.

Reply:
Done